cg2019_paper.tex

```diff
@@ -133,13 +134,13 @@ \section{Introduction}
 controller parameters better than a hand-made design, but at the same
 time revealed several challenges. In general, evolutionary algorithms
 optimize the fitness function that is used; evolved fuzzy controllers
-(hereafter FCs) will be eventually as good as the fitness function allows. 
+(hereafter FCs) will \textcolor{red}{eventually be} as good as the fitness function allows. 
 But in this particular case we cannot use as fitness function the position
 obtained by the FC in every possible race on every possible track with
 every possible opponent, so we have to settle for a {\em surrogate} of
 the fitness in a very limited environment. First we opted for
 eliminating opponents and making evaluations in solo races; then we
-chose a particular track that combined straight segments as well as
+\textcolor{red}{chosed} a particular track that combined straight segments as well as
```

Comment:
Esto no sería chose? Choose, chose...

Reply:
You are totally right. My bad! I got seriously confused.